diff -uNr lapack-3.7.0.orig/CMakeLists.txt lapack-3.7.0/CMakeLists.txt
--- lapack-3.7.0.orig/CMakeLists.txt	2017-02-07 01:17:52.294937516 +0300
+++ lapack-3.7.0/CMakeLists.txt	2017-02-07 01:18:17.575188367 +0300
@@ -1,13 +1,5 @@
 cmake_minimum_required(VERSION 2.8.10)
 
-# Set a default build type if none was specified
-if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
-  message(STATUS "Setting build type to 'Release' as none was specified.")
-  set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
-  # Set the possible values of build type for cmake-gui
-  set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
-endif()
-
 project(LAPACK Fortran)
 
 set(LAPACK_MAJOR_VERSION 3)
